projects/plugins/jetpack/modules/sso.php

```diff
@@ -27,1258 +11,17 @@
-		add_action( 'login_init', array( $this, 'login_init' ) );
-		add_filter( 'jetpack_xmlrpc_methods', array( $this, 'xmlrpc_methods' ) );
-		add_action( 'init', array( $this, 'maybe_logout_user' ), 5 );
-		add_action( 'jetpack_modules_loaded', array( $this, 'module_configure_button' ) );
-		add_action( 'login_form_logout', array( $this, 'store_wpcom_profile_cookies_on_logout' ) );
-		add_action( 'jetpack_unlinked_user', array( 'Jetpack_SSO_Helpers', 'delete_connection_for_user' ) );
 
-		add_action( 'jetpack_site_before_disconnected', array( static::class, 'disconnect' ) );
-		add_action( 'wp_login', array( 'Jetpack_SSO', 'clear_cookies_after_login' ) );
+use Automattic\Jetpack\Connection\SSO;
 
-		// Adding this action so that on login_init, the action won't be sanitized out of the $action global.
-		add_action( 'login_form_jetpack-sso', '__return_true' );
+SSO::get_instance();
 
-		add_filter( 'wp_login_errors', array( $this, 'sso_reminder_logout_wpcom' ) );
-
-		/**
-		 * Filter to include Force 2FA feature.
-		 *
-		 * By default, `manage_options` users are forced when enable. The capability can be modified
-		 * with the `jetpack_force_2fa_cap` filter.
-		 *
-		 * To enable the feature, add the following code:
-		 * add_filter( 'jetpack_force_2fa', '__return_true' );
-		 *
-		 * @param bool $force_2fa Whether to force 2FA or not.
-		 *
-		 * @todo Provide a UI to enable/disable the feature.
-		 *
-		 * @since 12.7
-		 * @module SSO
-		 * @return bool
-		 */
-		if ( ! class_exists( 'Jetpack_Force_2FA' ) && apply_filters( 'jetpack_force_2fa', false ) ) {
-			// Checking for the class to avoid collisions with existing standalone Jetpack Force 2FA plugin and break out if so.
-			require_once JETPACK__PLUGIN_DIR . 'modules/sso/class-jetpack-force-2fa.php';
-			new Jetpack_Force_2FA();
-		}
-
-		/*
-		 * Allow admins to invite new users to create a WordPress.com account
-		 * as they are added to the site.
-		 *
-		 * This is a feature that is only available when the admin is connected to WordPress.com.
-		 */
-		if (
-			Jetpack_SSO_Helpers::is_user_connected() &&
-			! is_multisite() &&
-			/**
-			 * Toggle the ability to invite new users to create a WordPress.com account.
-			 *
-			 * @module sso
-			 *
-			 * @since 13.3
-			 *
-			 * @param bool true Whether to allow admins to invite new users to create a WordPress.com account.
-			 */
-			apply_filters( 'jetpack_sso_invite_new_users_wpcom', true )
-		) {
-			new Jetpack_SSO_User_Admin();
-		}
-	}
-
-	/**
-	 * Returns the single instance of the Jetpack_SSO object
-	 *
-	 * @since 2.8
-	 * @return Jetpack_SSO
-	 **/
-	public static function get_instance() {
-		if ( self::$instance !== null ) {
-			return self::$instance;
-		}
-
-		self::$instance = new Jetpack_SSO();
-		return self::$instance;
-	}
-
-	/**
-	 * Add configure button and functionality to the module card on the Jetpack screen
-	 **/
-	public static function module_configure_button() {
+add_action(
+	'jetpack_modules_loaded',
```

Comment:
This is mostly only useful to show a configure link in the old module list:

<img width="845" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/f94c567f-f0ca-4534-8c95-bbd4dd83cc1b">